proto/osmosis/superfluid/tx.proto

```diff
@@ -54,6 +58,17 @@ message MsgSuperfluidUnbondLock {
 }
 message MsgSuperfluidUnbondLockResponse {}
 
+message MsgSuperfluidUndelegateAndUnbondLock {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  uint64 lock_id = 2;
+  // Amount of unlocking coin.
+  cosmos.base.v1beta1.Coin coin = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
```

Comment:
i dont think castrepeated is necessary when it's just a single coin, i've always seen people use cast repeated with "repeated" key term

```suggestion
   cosmos.base.v1beta1.Coin token_out = 3 [
    (gogoproto.moretags) = "yaml:\"coin\"",
    (gogoproto.nullable) = false
  ];
```